packages/client/hmi-client/src/types/Types.ts

```diff
@@ -229,6 +229,38 @@ export interface ModelFramework extends TerariumAssetThatSupportsAdditionalPrope
     semantics: string;
 }
 
+export interface InitialSemantic extends Semantic {
+    target: string;
+    expression: string;
+    modelConfiguration: ModelConfiguration;
+}
```

Comment:
I don't quite understand how the Java types are resolved here. 

Are you saying that in Javascript land we will have the full model-configuration object embedded into every children objects?

Also when we have `expression` we would normally also have `expression_mathml`, is this missed or are they somewhere else?

Reply:
For the `expression_mathml` point, I think initially I had an idea in my head of us being able to convert the expression to a mathml statement in the back end when we create an AMR from a ModelConfiguration.  

If that's not possible, maybe we have to add that key in for the InitialSemantic.

Reply:
@blanchco can you elaborate a bit more? What library do you have in mind to do the mathml conversion and do we know what kind of coverage we would get?

Reply:
I haven't looked into any libraries haha, might be be just to stick the `expression_mathml` key in there for now

Reply:
@blanchco Downstream processes use mathml, we cannot just leave it as a dummy field.

Reply:
I thought they all used SimPy for expression!?
Then, indeed we need to add that field back.